mappings/net/minecraft/block/IBlockInfoProxy.mapping

```diff
@@ -57,8 +57,13 @@ CLASS none/asl net/minecraft/block/IBlockInfoProxy
 		ARG 1 pos
 	METHOD e allowsLightTransmission ()Z
 	METHOD g getMapColor ()Lnone/ays;
-	METHOD h ()Z
+	METHOD h isFullBoundsCube2 ()Z
 	METHOD i getRenderType ()Lnone/api;
+	METHOD j ambientOcclusionLightLevel ()F
+	METHOD k isFullBoundsCubeForRendering ()Z
+	METHOD l isBasicBlock ()Z
 	METHOD m emitsRedstonePower ()Z
+	METHOD n ()Z
 	METHOD o getPistonBehavior ()Lnone/ayu;
-	METHOD q isOpaqueFullBoundsCube ()Z
+	METHOD p isFullBoundsCubeForSideCulling ()Z
+	METHOD q canPlaceThingsOn ()Z
```

Comment:
Most blocks can be placed an anything, this `canPlaceThingsOn` is only checked for the list of blocks I wrote out. I would like to come up with a good name for that category of blocks, perhaps.